lib/datasource/git-tags/index.ts

```diff
@@ -29,7 +32,7 @@ export async function getPkgReleases({
 
     return result;
   } catch (err) {
-    logger.debug({ err }, `Git-Tags lookup error in ${lookupName}`);
+    logger.error({ err }, `Git-Tags lookup error in ${lookupName}`);
```

Comment:
@viceice i cannot figure out how to run into this block since a potential null response from git-refs is now caught. i would want to avoid ignoring this from coverage if possible

Reply:
I think the catch can removed completely then